app/assets/stylesheets/admin/base.scss

```diff
@@ -0,0 +1,125 @@
+// General overrides for the active admin base styles.
+
+.site_title a {
+  color: white;
+
+  &:visited {
+    color: white;
+  }
+
+  &:hover {
+    text-decoration: none;
+    color: white;
+  }
+}
+
+.errors {
+  margin-top: 8px;
+  padding-top: 8px;
+}
+
+.handle.fa, .btn.fa {
+  font-size: 1.4rem;
+  vertical-align: middle;
+  margin-right: 0.6rem;
+}
+
+.inputs.reorderable label, .handle, .btn.fa {
+  cursor: pointer;
+}
+
+.has_many_container {
+  li {
+    float: left;
+    margin-bottom: 0;
+    padding-bottom: 0;
+  }
+
+  .has_many_delete,
+  .has_many_remove,
+  .handle {
+    margin-top: 40px;
+  }
+
+  .handle {
+    padding-top: 10px;
+  }
+
+  .has_many_delete label > input {
+    top: 2px;
+  }
+}
+
+// Minimum column widths
+.col-actions {
+  min-width: 78px;
+
+  a {
+    display: inline-block;
+  }
+}
+
+.col-id {
+  min-width: 47px;
+}
+
+.sidebar_section {
+  .select2 {
+    width: 100%;
+  }
+}
+
+.icon-gallery + .select2 .select2-selection {
+  height: initial;
+
+  img {
+    display: block;
+  }
+}
+
+.show_responses {
+  h2 {
+    margin-bottom: 0.5rem;
+  }
+}
+
+table.histogram {
+  .frequent {
+    font-weight: bold;
+  }
+
+  td {
+    padding-right: 2rem;
+
+    &:last-child {
+      min-width: 240px;
+      display: flex;
+    }
+  }
+
+  .bar {
+    display: inline-block;
+    background-color: #386165;
+    margin-right: 0.3rem;
+    height: 1rem;
+    vertical-align: middle;
+  }
+}
+
+.btn {
+  @extend input[type="submit"];
+}
+
+.btn:hover {
+  background-color: rgb(0, 201, 182);
+}
+
+#utility_nav:hover > li {
+    clip: rect(0, 0, 0, 0);
```

Comment:
[`clip`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip) doesn't seem work anymore (in Chrome anyway). These items are invisible, but since it's just an opacity trick, they're still the target for click events. According to MDN, we should use [`clip-path`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path) instead.